libsnark/gadgetlib1/gadgets/fields/fp2_gadgets.hpp

```diff
@@ -50,11 +50,13 @@ class Fp2_variable : public gadget<typename Fp2T::my_Fp> {
 
     void generate_r1cs_equals_const_constraints(const Fp2T &el);
     void generate_r1cs_witness(const Fp2T &el);
-    Fp2T get_element();
+    Fp2T get_element() const;
 
     Fp2_variable<Fp2T> operator*(const FieldT &coeff) const;
+    Fp2_variable<Fp2T> operator*(const Fp2T &fp2_const) const;
```

Comment:
`fp2_const` -> `other`?

Reply:
Hapy to change the name, but `other` kind of implies multiplication by another object of the same type.  Here we are multiplying by a constant Fp2 element using just linear combinations, in contrast to the multiplication of 2 variables like in `fp2_mul_gadget`.

Reply:
> other kind of implies multiplication by another object of the same type

I see, that's a good point. Let's keep it like that then.